staynomad/Nomad-Front
# State Dropdown && Confirm Search

# Relevant issue
Closes #531 
closes #524 

# Summary of change
- Made the state options drop-down menu conditionally render the states only if the country "United States" was selected.
- Also removed the state name if the country name is changed afterwards
- Now, after selecting the filter options and selecting "confirm" the new listings are populated and the URL is refreshed.

# Testing/Verification
- Select any country besides the "United States" and check if the options for the states appear (There should be none)
- Select "United States" then select the "states" dropdown menu and see if the states appear.
- After selecting a state, try changing the country and see if the state name disappears.
- Test out different filter options and click confirm. Then see if the respective listings show up.
